### PR TITLE
Bug cole fix attendance committee issues

### LINKED ISSUE #5: ApplicationComitteController rubo-cop
In app/controllers/attendance_committee_controller.rb

Line 8: `@committeeRows` should be snake_case (`@committee_rows`), Remove the semicolon
Line 10: `committeeList` should be snake_case (`committee_list`)
Line 13: `new_commitee` misspelled (`new_committee`)
Line 39: `@committeeAttendanceLogs` should be snake_case (`@committee_attendance_logs`)

Whole File: Fix tabbing, 2 spaces for each tab

### LINKED ISSUE #7: AttendanceCommitteeController doc comments
In app/controllers/attendance_committee_controller.rb

All functions need doc comments

### LINKED ISSUE #8: AttendanceCommitteeController before_action
In app/controllers/attendance_committee_controller.rb

Use before_action to check user sign-in and level. An example can be found in app/controllers/events_controller.rb

### PR DESCRIPTION
**Pull request by cdconn00**
Fixes #5 
Fixes #7 
Fixes #8

### CODE REVIEW

File: app/controllers/attendance_committee_controller.rb [hunk 1]
```diff
@@ -1,47 +1,45 @@
...
+    before_action only: [:index] do
+      confirm_logged_in('Please Log In')
+    end
+    before_action :confirm_staff, only: [:index, :show]
+
     class CommitteePoints
         attr_accessor :committee, :social_points, :fundraising_points, :campus_relations_points, :pr_points, :community_outreach_points, :give_back_points
       end
 
     def index
```
Comment (Buzzec) [inline app/controllers/attendance_committee_controller.rb:12]: Add comments before defs saying what this route accomplishes

**Review by Buzzec — CHANGES_REQUESTED** [review]

File: app/controllers/attendance_committee_controller.rb [hunk 2]
```diff
@@ -1,47 +1,45 @@
...
-                new_commitee.pr_points = Committee.point_total(com.id, "pr")[0].total_points
-                new_commitee.community_outreach_points = Committee.point_total(com.id, "community_outreach")[0].total_points
-                new_commitee.give_back_points = Committee.point_total(com.id, "give_back")[0].total_points
-
-                @committeeRows.push new_commitee
-            end
-        else
-            redirect_to root_path, alert: "You do not have permissions"
+        # Shows points for all committees as index, only >exec
+        @committee_rows = [];
```
Comment (Buzzec) [inline app/controllers/attendance_committee_controller.rb:14]: Remove semicolon

Comment (cdconn00) [reply app/controllers/attendance_committee_controller.rb]: Change with 3e30aff

File: app/controllers/attendance_committee_controller.rb (hunk 1 above)
Comment (cdconn00) [reply app/controllers/attendance_committee_controller.rb:13]: Change with 3e30aff

**Review by Buzzec — APPROVED** [review]

**Pull request merged**